bin/drush-install.sh

```diff
@@ -35,6 +35,31 @@ org.civicrm.contactsummary,\
 org.civicrm.bootstrapcivicrm,\
 org.civicrm.bootstrapcivihr
 
+## List of Default Enabled Cron Jobs
+ENABLED_JOBS=( "CiviCRM Update Check" \
```

Comment:
we have hrcore extension now and we could do such things in it , I see no reason to do it in drush-install.sh ... let's keep this sh file for things that cannot be done in using an extension ..

also if you moved this to hrcore , can you do a single array that contain the list of cron jobs along with its desired status in kay=>value pair like this : 

```
$cronJobs = [
'Clean-up Temporary Data and Files' => 1,
'Clone Documents' => 0,
etc ..
]
```

in then you can just loop through the result and set is_active value .